src/main/java/com/stripe/model/Event.java

```diff
@@ -14,15 +16,55 @@
 @Setter
 @EqualsAndHashCode(callSuper = false)
 public class Event extends ApiResource implements HasId {
-  @Getter(onMethod = @__({@Override})) String id;
-  String object;
+  @SerializedName("account")
```

Comment:
This should have a documentation right?

Reply:
I made a pay-sever PR 142016 with `account` documentation for OpenAPI. We can consider including the doc as
```
For Connect webhook events, the identifier of the account that emitted the event.
```
I modified the doc from .NET here
https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Entities/Events/Event.cs#L23
but emphasized that it's only on webhook but not events from endpoint

Reply:
👍  Fixed.